scripts/release/build/stage/upload/run.sh

```diff
@@ -4,13 +4,17 @@
 
 CHANNEL="$1"
 BUCKET_LOCATION="$2"
-FULLVERSION="$4"
 INSTANCE=$(cat scripts/release/common/ec2/tmp/instance)
 
-rm -rf pkg/* && mkdir -p pkg/"$FULLVERSION"
+scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r ubuntu@"$INSTANCE":~/fullversion.dat .
```

Comment:
nit: instead of copying it locally and deleting it afterward, just ssh and have it `cat ~/fullversion.dat`

Reply:
Good call, I'll change it.